app/assets/stylesheets/ama_layout/layout_components/breadcrumbs.scss

```diff
@@ -5,4 +5,16 @@
     display: inline-block;
     font-size: 0.75rem;
   }
+
+  a.disabled{
+    pointer-events: none;
+    color: $stormcloud;
+  }
+
+  li:not(:last-child)::after {
+    content: '>';
+    top: 0;
```

Comment:
Yea, by default foundation styles the separator with a `top: 1px;` which looks a little off when using `>` as a separator.